src/components/FlowRunAutomation.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div v-if="automationId" class="flow-run-automation">
+    <span>Automation</span> <AutomationIconText :automation-id />
+  </div>
+</template>
+
+<script setup lang="ts">
+  import { addMinutes, subMinutes } from 'date-fns'
+  import { computed } from 'vue'
+  import AutomationIconText from '@/components/AutomationIconText.vue'
+  import { useWorkspaceEvents } from '@/compositions/useWorkspaceEvents'
+  import { FlowRun } from '@/models/FlowRun'
+  import { getResourceIdParts } from '@/utilities/events'
+
+  const props = defineProps<{
+    flowRun: FlowRun,
+  }>()
+
+  const { events } = useWorkspaceEvents(() => {
+    if (!props.flowRun.expectedStartTime) {
+      return null
+    }
+
+    // purposefully adding some margin around the expectedStartTime
+    // because the automation being executed should happen prior to the time the run was scheduled
+    const until = addMinutes(props.flowRun.expectedStartTime, 5)
+    const since = subMinutes(until, 5)
+
+    return {
+      resource: {
```

Comment:
I think this would match automation created/updated events?

maybe should identify a particular event? automation.triggered or executed?

Reply:
Ah yes good catch, forgot to target the specific event I was looking at. Updated in https://github.com/PrefectHQ/prefect-ui-library/pull/2605/commits/051d0484be639fe6cef8e3ce3e39aa5268da3d2a